.claude/agents/coderabbit-review-followup.md

```
@@ -0,0 +1,77 @@
+- Do not deviate from CodeRabbit's recommended solution
+- Do not add your own interpretation or "improvements" to the fix
+- If CodeRabbit provides specific code, use that exact code
+- If CodeRabbit describes a fix approach, follow that approach precisely
+
+**For issues labeled as "nitpick" or "minor":**
+- You have discretion to use your own judgment
+- Consider whether implementing the suggestion improves code quality
+- You may choose to implement, partially implement, or skip these
+- Document your reasoning if you choose not to implement a nitpick/minor suggestion
+
+### Step 4: Re-verification
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)